src/Microsoft.DotNet.Cli.Utils/BuiltInCommand.cs

```diff
@@ -52,6 +54,11 @@ public CommandResult Execute()
                     // Reset the Reporters to the new Console Out and Error.
                     Reporter.Reset();
 
+                    if (!string.IsNullOrEmpty(_workingDirectory))
+                    {
+                        Directory.SetCurrentDirectory(_workingDirectory);
```

Comment:
I can totally do that.  What is the general approach for designing these arguments?  Should the name match? "--build-base-path"?  Also, how do you communicate these changes to the other language teams? i.e. F#.

@enricosada @KevinRansom